src/app/seamly2d/mainwindow.cpp

```diff
@@ -7161,20 +7164,20 @@ bool MainWindow::IgnoreLocking(int error, const QString &path)
             case QLockFile::LockFailedError:
                 answer = QMessageBox::warning(this, tr("Locking file"),
                                                tr("This file already opened in another window. Ignore if you want "
-                                                  "to continue (not recommended, can cause a data corruption)."),
+                                                  "to continue (!recommended, can cause a data corruption)."),
```

Comment:
I dont think you wanted to change the error message here

Reply:
No... it's also why I prefer to use !  as an op so as not to confuse when "not" is wanted.  :) 

And what are the chances  that "(not ...." was wanted.  Downside of search & replace all.  :(